source/guide_openslides.rst

```diff
@@ -0,0 +1,208 @@
+.. highlight:: console
+
+.. author:: GodMod <godmod@eqdkp-plus.eu>
+.. tag:: lang-python
+.. tag:: django
+.. tag:: presentations
+.. tag:: web
+.. tag:: assemblies
+.. tag:: meetings
+.. tag:: audience-business
+
+.. sidebar:: About
+
+  .. image:: _static/images/openslides.svg
+      :align: center
+
+##########
+OpenSlides
+##########
+
+.. tag_list::
+
+OpenSlides is a free, web-based presentation and assembly system for managing and projecting agenda, motions, and elections of assemblies.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Python <lang-python>` and its package manager pip
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`Web Backends <web-backends>`
+
+License
+=======
+
+OpenSlides_ is released under the `MIT License`_.
+
+Prerequisites
+=============
+
+Your URL needs to be setup:
+
+::
+
+ [isabell@stardust ~]$ uberspace web domain list
+ isabell.uber.space
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Step 1
```

Comment:
Please replace numbered steps with description of what is going on. The numbers do not add any information and make it cumbersome to edit the guide later.